packages/core/src/config.js

```diff
@@ -244,6 +253,13 @@ export const snapshotSchema = {
         domTransformation: { $ref: '/config/snapshot#/properties/domTransformation' },
         enableLayout: { $ref: '/config/snapshot#/properties/enableLayout' },
         reshuffleInvalidTags: { $ref: '/config/snapshot#/properties/reshuffleInvalidTags' },
+        scopeOptions: {
```

Comment:
it would have same definition as L62 ( make it similar to above lines)

Reply:
It is same, and this is how it is defined for other properties